README.md

```diff
@@ -1 +1,123 @@
+- [Autopilot-MySQL](https://github.com/autopilotpattern/mysql/): we're using the Autopilot-MySQL project to leverage the great features built into this container. It users MySQL5.6 via [Percona Server](https://www.percona.com/software/mysql-database/percona-server), and [`xtrabackup`](https://www.percona.com/software/mysql-database/percona-xtrabackup) for running hot snapshots.
+- [Memcached](https://github.com/autpilotpattern/memcached/): Caches often accessed data in memory so WordPress doesn't always have to access the database
+- [Nginx](https://github.com/autopilotpattern/nginx): Front-end load balancer for the WordPress envrionment, passes traffic from users to the WordPress containers on the back-end.
+- [NFS](https://github.com/autpilotpattern/nfsserver/): Stores user uploaded files so these files can be shared between many WordPress containers and served to the world
+- [Consul](https://www.consul.io/): used to coordinate replication and failover
+- [Manta](https://www.joyent.com/object-storage): the Joyent object store, for securely and durably storing our MySQL snapshots.
+
+### How do I use this thing?
+
+Clone this repository and place the WordPress theme you want to use into the `var/www/html/content/themes` directory. Follow the instructions below to set up your system to run containers on [Triton](https://www.joyent.com/), or anywhere that supports the Autopilot Pattern.
+
+### Getting started on Triton
+
+1. [Get a Joyent account](https://my.joyent.com/landing/signup/) and [add your SSH key](https://docs.joyent.com/public-cloud/getting-started).
+1. Install the [Docker Toolbox](https://docs.docker.com/installation/mac/) (including `docker` and `docker-compose`) on your laptop or other environment, as well as the [Joyent Triton CLI](https://www.joyent.com/blog/introducing-the-triton-command-line-tool) (`triton` replaces our old `sdc-*` CLI tools)
+1. [Configure Docker and Docker Compose for use with Joyent](https://docs.joyent.com/public-cloud/api-access/docker):
+
+```bash
+curl -O https://raw.githubusercontent.com/joyent/sdc-docker/master/tools/sdc-docker-setup.sh && chmod +x sdc-docker-setup.sh
+./sdc-docker-setup.sh -k us-east-1.api.joyent.com <ACCOUNT> ~/.ssh/<PRIVATE_KEY_FILE>
+```
+### Configure your envrionment
+
+Check that everything is configured correctly by running `./setup.sh`, you will need to pass the path of the ssh key you configured on your Joyent account with the `-k` flag. If it returns without an error you're all set. This script will create and `_env` file that includes the variables that you will need to run your WordPress envrionment.
+
+### Environment
+
+There are several sections in the _env file that you will need to configure for your specific environment.
+
+#### MySQL settings
+
+```
+# Environment variables for MySQL service
+MYSQL_DATABASE=test
+MYSQL_USER=wpuser
+MYSQL_PASSWORD=wppass
```

Comment:
This and other passwords are now pre-set set to random strings, though users can change them as they desire.